src/Data/Managers/V2/helpers/useHelpers.ts

```diff
@@ -0,0 +1,42 @@
+import { useContext } from "react";
+import { DependencyContext } from "@/UI";
+
+export const useHelpers = () => {
```

Comment:
useHelper is not specific enough in this case. These are helpers that are meant to help with http requests. I'd opt for a better naming such as `useHttpHelpers` or something similar.

Reply:
I would put in the name what it is a helper for. The "use" part I also do not get but that might be something that is specific to the js/ts ecosystem.

As far as I can tell this is a helper for `fetch` so I would name this `fetchHelpers` or if the `use` has a meaning: `useFetchHelpers`

Reply:
@bartv  the 'use' keyword is part of the react ecosystem and are used to identify hooks.

> Hooks are defined using JavaScript functions, but they represent a special type of reusable UI logic with restrictions on where they can be called.

Based on the definition of hooks, this helper doesn't need to be considered one I believe. `httpHelpers` or `fetchHelpers` might be more accurate.